### DISCUSSION TITLE
Welcome to laraflow Discussions!

### DISCUSSION THREAD

**Discussion by szana8** [szana8/Laraflow#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nlmansilla** [comment]
Hi colleagues. Will you add support for Laravel > 10?
Thanks!

**Reply by szana8** [reply]
Sure, I will try to add a Laravel 10+ support in this week.
